src/networking.c

```diff
@@ -2417,8 +2415,10 @@ void clientCommand(client *c) {
 "    Return information about client connections. Options:",
 "    * TYPE (NORMAL|MASTER|REPLICA|PUBSUB)",
 "      Return clients of specified type.",
-"PAUSE <timeout>",
-"    Suspend all clients for <timout> milliseconds.",
+"UNPAUSE",
+"    Stop the current client pause, resuming traffic."
```

Comment:
this one is missing a comma, right?
```suggestion
"    Stop the current client pause, resuming traffic.",
```

Reply:
yup, C just happily concatenated them :)